src/components/List.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+import cn from 'classnames';
+import debounce from 'lodash.debounce';
+import { Person } from '../types/Person';
+
+type Props = {
+  people: Person[],
+  delay: number,
+  onSelect: (person: Person | null) => void,
+};
+
+export const List: React.FC<Props> = ({ people, delay, onSelect }) => {
+  const [query, setQuery] = useState('');
+  const [appliedQuery, setAppliedQuery] = useState('');
+  const [visibleList, setVisibleList] = useState(false);
+
+  const applyQuery = React.useCallback(
+    debounce(setAppliedQuery, delay),
+    [],
+  );
+
+  const filteredPeople = React.useMemo(() => {
+    return people.filter(el => el.name.toLowerCase()
+      .includes(appliedQuery.toLowerCase()));
```

Comment:
- code is not aligned
- use need to provide more descriptive name that `el`
check other places 
```suggestion
    return people.filter(person => 
      person.name.toLowerCase().includes(appliedQuery.toLowerCase())
    );
```

Reply:
> * code is not aligned
![image](https://github.com/mate-academy/react_autocomplete/assets/106952290/3a4edc53-a7a7-486d-9ea5-3c94bc398f77)
Like this?